spec/models/ems_refresh/refreshers/openstack_infra_refresher_rhos_juno_spec.rb

```diff
@@ -59,8 +59,8 @@ def assert_table_counts
     Snapshot.count.should                    == 0
     Switch.count.should                      == 0
     SystemService.count.should               == 0
-    Relationship.count.should                == 0
-    MiqQueue.count.should                    == 8
+    Relationship.count.should                == 135
```

Comment:
@blomquisg weird, it's like these records are being accumulated in other tests? Or why are they changed?